src/main.cpp

```diff
@@ -82,94 +76,47 @@ void deinit()
 void setupPHYSFS()
 {
 	FileSystem& fs = FileSystem::getSingleton();
-	const std::string separator = fs.getDirSeparator();
+	
 	// Game should be playable out of the source package on all
 	// relevant platforms.
 	#if BLOBBY_ON_DESKTOP
-		std::string baseSearchPath("data" + separator);
-	#elif (defined __ANDROID__)
-		std::string baseSearchPath(SDL_AndroidGetExternalStoragePath() + separator);
-	#elif (defined __APPLE__) || (defined __SWITCH__)
-		// iOS
-		std::string baseSearchPath(PHYSFS_getBaseDir());
+		std::string baseSearchPath("data" + fs.getDirSeparator());
+	#else
+		std::string baseSearchPath(fs.getBaseDir());
```

Comment:
is this guaranteed to return a  path with dir separator at the end? Couldn't find anything on this in the docs, but if so, maybe put a comment here to explain the difference with the call aboce

Reply:
To make sure we have no issues in future I added a simple join method.

Reply:
great idea